src/components/MessagesList/MessagesGroup/Message/MessagePart/Mention.vue

```diff
@@ -96,12 +98,8 @@ export default {
 			return this.isGroupMention
 				&& loadState('spreed', 'user_group_ids', []).includes(this.id)
 		},
-		isDarkTheme() {
-			return window.getComputedStyle(document.body)
-				.getPropertyValue('--background-invert-if-dark') === 'invert(100%)'
-		},
 		avatarUrl() {
-			return generateOcsUrl('apps/spreed/api/v1/room/{token}/avatar' + (this.darkTheme ? '/dark' : ''), {
+			return generateOcsUrl('apps/spreed/api/v1/room/{token}/avatar' + (isDarkTheme ? '/dark' : ''), {
```

Comment:
Fixed this file for chat mentions:
![image](https://github.com/nextcloud/spreed/assets/93392545/19f6f24e-9866-4c49-8ad8-35992d821a8f)

Because of the wrong naming, it wasn't working before, so not your fault to miss it =)